libraries/Microsoft.Bot.Connector/Authentication/EmulatorValidation.cs

```diff
@@ -70,6 +70,18 @@ public static bool IsTokenFromEmulator(string authHeader)
                 return false;
             }
 
+            var tid = token.Claims.FirstOrDefault(claim => claim.Type == "tid")?.Value != null ? '/' + token.Claims.FirstOrDefault(claim => claim.Type == "tid")?.Value + '/' : string.Empty;
+
+            //Validate if there is an existing issuer with the same tid value.
+            if (!string.IsNullOrEmpty(tid) && !ToBotFromEmulatorTokenValidationParameters.ValidIssuers.Any((issuer) => issuer.Contains(tid)))
+            {
+                //If the issuer doesn't exist, this is added using the Emulator token issuer structure.
+                //This allows use of the SingleTenant authentication through Emulator.
+                var newIssuer = "https://sts.windows.net" + tid;
```

Comment:
This url can be created with this `ValidTokenIssuerUrlTemplateV1` located in the AuthenticationConstants file.